### PR TITLE
[Event Hubs Client] October Changelog Updates

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The focus of these changes is to update the change logs for each Event Hubs client package to detail the changes in the forthcoming release for the 
October milestone.

# Last Upstream Rebase

Friday, September 25, 4:47pm (EDT)

### CODE REVIEW

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-## 5.3.0-beta.2 (Unreleased)
+## 5.3.0-beta.2 (2020-09-29)
+
+### Changes
+
+#### New Features
+
+- The `EventData` representation has been extended with the ability to treat the `Body` as `BinaryData`, providing support for a variety of data transformations and `ObjectSerializer` integration.
+
+- `EventData` has been integrated with the new Schema Registry service, via use of the `ObjectSerializer` with `BinaryData`.
```
Comment (MiYanni) [inline sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md:11]: I'd go with something more like:
> Accessing `Body` as `BinaryData` allows the ability to provide serialization logic when sending or receiving events. Any type that derives of `ObjectSerializer`, such as `JsonObjectSerializer` and `SchemaRegistryAvroObjectSerializer`, can utilize this workflow. The aforementioned Avro serializer utilizes the Azure Schema Registry service.

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+**Note:** The Schema Registry feature is also a new addition to the service and may not yet be available in all regions or Azure clouds.
```
Comment (MiYanni) [inline sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md:13]: I believe the Schema Registry is being marketed as a separate service, even though it is in the EventHubs realm. So, I don't know if calling it a 'feature' is right(?) The high-level summary of it is:
> Azure Schema Registry is a hosted schema repository service provided by Azure Event Hubs, designed to simplify schema management and data governance. 

So, maybe changing the wordage of this note would help it be more accurate.

File: sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
-## 5.3.0-beta.2 (Unreleased)
+## 5.3.0-beta.2 (2020-09-29)
+
+### Changes
+
+#### New Features
+
+- The `EventData` representation has been extended with the ability to treat the `Body` as `BinaryData`, providing support for a variety of data transformations and `ObjectSerializer` integration.
+
+- `EventData` has been integrated with the new Schema Registry service, via use of the `ObjectSerializer` with `BinaryData`.
```
Comment (MiYanni) [inline sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md:11]: See comment from other changelog.

**Review by MiYanni — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md (hunk 2 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md:13]: According to the preview announcements, they're marketing it as a part of Event Hubs:

- [Public Preview of the Azure Schema Registry in Azure Event Hubs](https://techcommunity.microsoft.com/t5/messaging-on-azure/public-preview-of-the-azure-schema-registry-in-azure-event-hubs/ba-p/1699878)
- [Azure Schema Registry in Event Hubs (Preview)](https://docs.microsoft.com/en-us/azure/event-hubs/schema-registry-overview)

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md (hunk 1 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md:11]: I like that.   Thanks!

Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md:11]: Gave it a bit of wordsmithing, but kept the theme.

File: sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md (hunk 2 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md]: Gave it a bit of wordsmithing to incorporate your suggestion, but kept the strong association with Event Hubs.

**Pull request merged**